sql/log.cc

```diff
@@ -7042,6 +7162,19 @@ Event_log::prepare_pending_rows_event(THD *thd, TABLE* table,
       flush the pending event and replace it with the newly created
       event...
     */
+    binlog_cache_mngr *cache_mngr= thd->binlog_get_cache_mngr();
+    
+    if (cache_mngr)
+    {
+      binlog_cache_data *cache_data= (cache_mngr->get_binlog_cache_data(is_transactional));
+      /*
+        If the ANNOTATE and TABLE_MAPS have been filtered earlier using binlog_dump_* then 
```

Comment:
```suggestion
        If all the ANNOTATE and TABLE_MAPS have been filtered earlier using binlog_dump_* then 
```

Also the line is too long